src/core/objects/object_mover.cpp

```diff
@@ -33,7 +33,11 @@
 namespace OpenOrienteering {
 
 ObjectMover::ObjectMover(Map* map, const MapCoordF& start_pos)
- : start_position(start_pos), prev_drag_x(0), prev_drag_y(0), constraints_calculated(true)
+ : start_position(start_pos)
+ , corner_tolerance(0)
+ , prev_drag_x(0)
+ , prev_drag_y(0)
+ , constraints_calculated(true)
```

Comment:
I would prefer to move the constant initializations to the member variable declaration.

Reply:
Me too. Assuming that you are talking about all the data members, I'll rather move the refactoring to a separate patch so that I don't mix unrelated changes into the feature implementation.